src/server/svr_movejob.c

```diff
@@ -919,10 +969,7 @@ send_job(job *jobp, pbs_net_t hostaddr, int port, int move_type,
 			set_job_substate(jobp, JOB_SUBSTATE_TRNOUTCM);
 		}
 
-		if (PBSD_rdytocmt(con, job_id, PROT_TCP, NULL) != 0)
-			continue;
-
-		if (PBSD_commit(con, job_id, PROT_TCP, NULL) != 0) {
+		if (PBSD_commit(con, job_id, PROT_TCP, NULL, NULL) != 0) {
```

Comment:
The PBS_BATCH_Rdytocommit message is deprecated. The server does not do anything with it anymore, but simply acks the request (in case some client makes this call)